ssl_test.go

```diff
@@ -0,0 +1,464 @@
+//go:build !go_tarantool_ssl_disable
```

Comment:
It would be nice to add an example with connection via TLS.

Reply:
Thank you, added. See example_test.go/ExampleSslOpts().

Reply:
Thanks!